spec/features/upload/v2/template.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe 'restoring comments' do
+  before { login_to_project_as_user }
+
+  context 'template with comments' do
+    before do
+      visit upload_path
+    end
+
+    it "transforms methodologies into boards", js: true do
+      select "Dradis::Plugins::Projects::Upload::Template"
+      attach_file \
+        "file",
+        Rails.root.join(
+          "spec", "fixtures", "files", "templates", "with_comments.xml"
+        )
+
+      expect(page).to have_text("Worker process completed", wait: 120)
```

Comment:
this `120` makes me thing about using something different from feature specs

Reply:
I just copied this from the `upload_spec` in Pro.

Reply:
> I just copied this from the upload_spec in Pro.

Is that a good enough reason to keep it?